src/main/java/androidx/content/Context.kt

```diff
@@ -91,3 +95,27 @@ inline fun Context.withStyledAttributes(
         typedArray.recycle()
     }
 }
+
+/**
+ * Return a [Bitmap] decoded from [Resources]
+ *
+ * @param resId Resource id of a drawable from `res/drawable` folder
+ * @param options null-ok; Options that control downsampling and whether the
+ * image should be completely decoded, or just is size returned.
+ */
+inline fun Resources.getBitmap(
+    @DrawableRes resId: Int,
```

Comment:
it would be cool to have inline classes in the future: `DrawableInt.getBitmap(resources)` 😸